src/main/kotlin/org/jitsi/nlj/Transceiver.kt

```diff
@@ -236,10 +236,10 @@ class Transceiver(
         payloadTypes.clear()
     }
 
-    fun addRtpExtension(extensionId: Byte, rtpExtension: RTPExtension) {
-        logger.cdebug { "Adding RTP extension: $extensionId -> $rtpExtension" }
-        rtpExtensions[extensionId] = rtpExtension
-        val rtpExtensionAddedEvent = RtpExtensionAddedEvent(extensionId, rtpExtension)
+    fun addRtpExtension(rtpExtension: RtpExtension) {
+        logger.cdebug { "Adding RTP extension: ${rtpExtension.id} -> ${rtpExtension.type}" }
```

Comment:
if we change extension to a data class we'll get a tostring for free and could just log the RtpExtension itself here `$rtpExtension`